packages/generator/src/edmx-to-vdm/v4/operation.ts

```diff
@@ -124,13 +124,18 @@ export function filterAndTransformOperations(
     operation => (operation.IsBound.toLowerCase() === 'true') === isBound
   );
 
-  const [withoutOperation, withOperation] = splitMissingOperation(
+  const [withoutOperation] = splitMissingOperation(
+    operationImports,
+    operations
+  );
+
+  const [, boundWithOperation] = splitMissingOperation(
```

Comment:
[req] I think it would be better to filter only the operations, that have an operation import. Otherwise you will get two logs for the same thing and this might be unnecessary.
[nth] You could consider switching the logic of `splitMissingOperation()` to return the operations with operation first, then line 132 would look slightly cleaner (and line 127 would be different with my previous comment anyways).

Reply:
I did switch the logic of `splitMissingOperation()` as you said.

> I think it would be better to filter only the operations, that have an operation import.

Do you mean adding something like this to filter out operation imports that have an operation separately?
```
const withoutOperation = operationImports.filter(
    operation => !findOperationByImportName(operations, operation.operationName)
);
```